### PR TITLE
Pretty entity inspect

### LINKED ISSUE #246: Display attributes for #to_s
Imagine I have following Entity:

```
class Institution
  include Lotus::Entity

  attributes :name, :code
end
```

when I instantiate a new instance:

```
irb> i = Institution.new(name: 'School of Art')
irb> i
=> #<Institution:0x007f82efabb2c8 @name ="School of Art'>
```

we could see that only assigned attributes are displayed. I think it'd be better if we display all attributes of the entity for inspection (even if they are nil):

So the output should be:

```
irb> i
=> #<Institution:0x007f82efabb2c8 @name ="School of Art' @code=nil>
```

Thoughts?

### PR DESCRIPTION
**Pull request by aderyabin**
The PR should resolve #246 .

### CODE REVIEW

File: lib/lotus/entity.rb [hunk 1]
```diff
@@ -223,7 +223,50 @@ def ==(other)
...
+    # @example
+    #   require 'lotus/model'
+    #   class User
+    #     include Lotus::Entity
+    #     attributes :name
+    #   end
+    #
+    #   user = User.new(id: 23, name: 'Luca')
+    #   user.attribute_names # #<Set: {:id, :name}>
+
```
Comment (joneslee85) [inline lib/lotus/entity.rb]: please remove empty line

File: lib/lotus/entity.rb [hunk 2]
```diff
@@ -223,7 +223,50 @@ def ==(other)
...
+    # @example
+    #   require 'lotus/model'
+    #   class User
+    #     include Lotus::Entity
+    #     attributes :name, :email
+    #   end
+    #
+    #   user = User.new(id: 23, name: 'Luca')
+    #   user.inspect # #<User:0x007fa7eefe0b58 @id=nil @name="Luca" @email=nil>
+
```
Comment (joneslee85) [inline lib/lotus/entity.rb]: please remove empty line

File: lib/lotus/entity.rb [hunk 3]
```diff
@@ -246,5 +289,13 @@ def update(attributes={})
       end
     end
 
+    private
+
+    # Return the value by attribute name
+    #
+    # @since 0.5.1
```
Comment (joneslee85) [inline lib/lotus/entity.rb]: please document with `@private`

**Comment by joneslee85** [comment]
thanks for PR, what about `#to_s`

**Comment by aderyabin** [comment]
@joneslee85. Thanks for code comments. Fixed.

**Pull request merged**